src/NHibernate.Config.Templates/HANA.cfg.xml

```diff
@@ -7,8 +7,11 @@ for your own use before compile tests in VisualStudio.
 <hibernate-configuration  xmlns="urn:nhibernate-configuration-2.2" >
 	<session-factory name="NHibernate.Test">
 		<property name="connection.driver_class">NHibernate.Driver.HanaDriver</property>
+		<!-- Sap.Data.Hana does not support re-enlisting in the same transaction, wrecking many usages
+		     with NHibernate when both NHibernate and the connection performs auto-enlistment. -->
 		<property name="connection.connection_string">
-			Server=localhost:39015;UserID=nhibernate;Password=
+			Server=localhost:39015;UserID=nhibernate;Password=;
+			Enlist=false;
```

Comment:
This is the way to fix most transaction scope tests failures in my opinion. So this needs to be adjusted in any previously existing test configuration.